app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -277,11 +290,15 @@ export default {
             weight: 60,
             components: {
               orgbook: true,
-              simplefile: this.userType !== this.ID_MODE.PUBLIC
+              simplefile: this.userType !== this.ID_MODE.PUBLIC,
             },
           },
         },
         templates: templateExtensions,
+        evalContext: {
```

Comment:
Did you find any good source of documentation on this?
I've seen it referenced a bunch in the past but never a good central explainer...

Reply:
Figured it out, this lets you see the result of what you set in Custom Default Value at design time 👍